### DISCUSSION TITLE
How to move a project from Tye to Aspire?

### DISCUSSION THREAD

**Discussion by Cotspheer** [microsoft/aspire#1807] [Q&A]
I just recognized that the last light went out for Project Tye. I have a project with many services including azure functions that we used to test locally by using tye as an orchestrator. They all use dapr as a bridge for sync communication. It was a convenient way to start all services at once and making sure they can communicate with each other.

Is there a migration path? Any fellow people in the same position? What were your solution? Is Aspire even the right tool?

Sorry for the questions but I hope for some answers to guide me in the right direction as reading the Aspire docs will take time.

**Comment by davidfowl** [comment]
There will be depending on what features you used:
- Azure functions - There's no native as yet and may not be for v1 (though [there are workarounds](https://github.com/dotnet/aspire-samples/issues/58)
- Dapr - [There's native dapr support](https://github.com/dotnet/aspire-samples/tree/main/samples/AspireWithDapr)

**Reply by Cotspheer** [reply]
Thank you! Will look into that!
